source/world/level/storage/ExternalFileLevelStorage.cpp

```diff
@@ -185,7 +187,7 @@ LevelChunk* ExternalFileLevelStorage::load(Level* level, int x, int z)
 	LevelChunk* pChunk = new LevelChunk(level, pData, x, z);
 	pBitStream->Read((char*)pChunk->m_tileData, 16 * 16 * 128 / 2);
 
-	if (m_pLevelData->getVersion() == 1)
+	if (m_pLevelData->getStorageVersion() == 1)
```

Comment:
If you changed L227 to check for >=1, please check the same here

Reply:
Good catch. This is fixed in my upcoming commit, I'll resolve this conversation once I push it.